### DISCUSSION TITLE
Ship weapons take EMP damage, inflicted onto the loaders

### DISCUSSION THREAD

**Discussion by Solbusaur** [FakeFishGames/Barotrauma#5841] [Ideas and suggestions]
If a coilgun/railgun etc. comes in range of an EMP explosion, the effect should be delivered to every linked loader that that gun has. This should make all PvP situations more convenient with larger ships where supercapacitors cannot be reached by EMP blasts.
![image](https://user-images.githubusercontent.com/63509011/118843148-ae747a00-b8c1-11eb-9a9d-c5104054d500.png)

**Comment by Udrakan** [comment]
Since loaders are mechanical, how about transfering EMP effect to all wired supercapacitors instead?

**Reply by Solbusaur** [reply]
Either could work. The problem arises when sub designers get dangerously creative and connect the guns to large capacity batteries or directly to the power grid.
Plus EMPs already damage loaders
